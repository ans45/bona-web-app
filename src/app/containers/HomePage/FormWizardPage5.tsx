import Box from '@material-ui/core/Box';
import { FormWizardPage } from 'app/components/FormWizard/FormWizardPage';
import { FormWizardPageTitle } from 'app/components/FormWizardPageTitle';
import { Tiles } from 'app/components/Tiles';
import React from 'react';

export const FormWizardPage5 = () => (
  <FormWizardPage validate={() => undefined} submitButtonText="Submit">
    <Box mb={4}>
      <FormWizardPageTitle>Center aligned question?</FormWizardPageTitle>
      <Tiles
        name="tilesInput"
        data={[
          {
            label: 'Yes',
            sublabel: 'Lorem ipsum dolor. Ipsum dolor lorem.',
            value: 'yes',
          },
          {
            label: 'No',
            sublabel: 'Lorem ipsum dolor. Ipsum dolor lorem.',
            value: 'no',
          },
        ]}
      />
    </Box>
  </FormWizardPage>
);
