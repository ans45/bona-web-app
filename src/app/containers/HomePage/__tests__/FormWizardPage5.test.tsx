import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPage5 } from '../FormWizardPage5';
import { Form } from 'react-final-form';

describe('<FormWizardPage5  />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Form onSubmit={() => undefined}>{() => <FormWizardPage5 />}</Form>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
