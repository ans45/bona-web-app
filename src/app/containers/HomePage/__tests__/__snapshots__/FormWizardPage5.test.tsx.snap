// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPage5  /> should match snapshot 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
>
  <div
    class="MuiBox-root MuiBox-root-4"
  >
    <h2
      class="MuiTypography-root makeStyles-root-2 makeStyles-root-3 MuiTypography-h5"
    >
      Center aligned question?
    </h2>
  </div>
  <div
    class="MuiFormControl-root makeStyles-formControl-5"
  >
    <div
      class="MuiFormGroup-root makeStyles-radioGroup-6"
      role="radiogroup"
    >
      <div
        class="MuiPaper-root makeStyles-root-7 makeStyles-root-10 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <p
          class="MuiTypography-root makeStyles-label-8 MuiTypography-subtitle1"
        >
          Yes
        </p>
        <p
          class="MuiTypography-root makeStyles-sublabel-9 MuiTypography-subtitle2"
        >
          Lorem ipsum dolor. Ipsum dolor lorem.
        </p>
      </div>
      <div
        class="MuiPaper-root makeStyles-root-7 makeStyles-root-11 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <p
          class="MuiTypography-root makeStyles-label-8 MuiTypography-subtitle1"
        >
          No
        </p>
        <p
          class="MuiTypography-root makeStyles-sublabel-9 MuiTypography-subtitle2"
        >
          Lorem ipsum dolor. Ipsum dolor lorem.
        </p>
      </div>
    </div>
  </div>
</div>
`;
